PWGHF/Core/CentralityEstimation.h

```diff
@@ -113,6 +113,88 @@ float getCentralityColl(const Coll&)
   return 105.0f;
 }
 
+/// Get the centrality
+/// \param collision is the collision with the centrality information
+/// \param centEstimator integer to select the centrality estimator
+/// \return collision centrality
+template <typename Coll>
+float getCentralityColl(const Coll& collision, int centEstimator)
+{
+  float cent = -999.;
+  switch (centEstimator) {
+    case CentralityEstimator::FT0A:
+      if constexpr (hasFT0ACent<Coll>) {
+        cent = collision.centFT0A();
+      } else {
+        LOG(warning) << "Warning: Collision does not have centFT0A().\n";
+      }
+      break;
```

Comment:
Make the code fail as soon as a critical error occurs.
```suggestion
  switch (centEstimator) {
    case CentralityEstimator::FT0A:
      if constexpr (hasFT0ACent<Coll>) {
        return collision.centFT0A();
      }
      LOG(fatal) << "Collision does not have centFT0C().";
      break;
```